index.js

```diff
@@ -69,13 +72,32 @@ const getCompiledFileSync = (moduleDir, path) => {
   return transformSrc(moduleDir, res.toString())
 }
 
+const isInSourceBuild = moduleOptions => {
```

Comment:
Can we use a Webpack config option instead like we do with the `module` field? We can switch both the reading the `bsconfig.json` at a later time